crates/hyperdrive-math/src/short/max.rs

```diff
@@ -754,8 +747,8 @@ mod tests {
 
     #[tokio::test]
     async fn fuzz_sol_calculate_max_short_without_budget_then_open_short() -> Result<()> {
-        let max_base_tolerance = fixed!(10e18);
-        let max_bonds_tolerance = fixed!(1e2);
+        let max_bonds_tolerance = fixed!(1e10);
```

Comment:
Why did the bond tolerance go up?

Reply:
I was doing extensive fuzz testing on the `main` branch and found that `1e2` was not sufficient. So the bump is not directly due to this PR.

Reply:
That being said, I think this is probably worth investigating further to see if it is a problem with the test or a problem with the code.